src/jitlog/build.lua

```diff
@@ -0,0 +1,46 @@
+local stdout = io.stdout
+local args = {...}
+
+if args[1] == "GC64" then
+  GC64 = true
+end
+
+--Work around limited api in when run under minilua
```

Comment:
Comment doesn't parse (and "API" should be capitalised thus).

Reply:
fixed